operator/main.go

```diff
@@ -213,6 +214,8 @@ func defineFlagVar() *FlagVar {
 		"The name of the Istio virtual service to be updated.")
 	flag.StringVar(&flagVar.vsNamespace, "virtual-svc-namespace", metav1.NamespaceDefault,
 		"The namespace of the Istio virtual service to be updated.")
+	flag.StringVar(&flagVar.istioGateway, "istio-gateway", "default/kyma-gw",
```

Comment:
do we really need these value to be configured as flag? what's the purpose to introduce these 3 overwrites?